src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1865,6 +1870,47 @@ ErrorIfQueryNotSupported(Query *queryTree)
 }
 
 
+/*
+ * VerifyDistinctClauseSupported returns true is provided query has supported
+ * distinct clause. Only target entries with Var, Const, Aggref types are supported
+ * at the moment.
+ * TODO : function support should be evaluated. immutable or stable functions can
```

Comment:
What is supported and what is not supported? Could you be a bit more descriptive?